packages/app-store/routing-forms/pages/route-builder/[...appPages].tsx

```diff
@@ -167,6 +167,9 @@ const buildEventsData = ({
     });
 
     group.eventTypes.forEach((eventType) => {
+      if (eventType.teamId && eventType.schedulingType === SchedulingType.MANAGED) {
+        return;
+      }
```

Comment:
A parent-managed event type always shows no availability, as it cannot be booked